atd-vze/src/views/Crashes/CrashChangeLog.js

```diff
@@ -0,0 +1,216 @@
+        }
+
+        // For each entry created in the diff array, generate an HTML table row.
+        let modalItems = diff.map(item => (
+            <tr key={record.id}>
+                <td>
+                    {item.original_record_key}
+                </td>
+                <td>
+                    <Badge color="primary">{String(item.original_record_value)}</Badge>
+                </td>
+                <td>
+                    <Badge color="danger">{String(item.archived_record_value)}</Badge>
+                </td>
+            </tr>
+        ));
+
+        // Generate the body of the modal box
+        modalBody = <section>
+            <h6>Crash ID: <Badge color={"secondary"} className={"float-right"}>{record.record_crash_id}</Badge></h6>
+            <h6>Archive Date: <Badge color={"secondary"} className={"float-right"}>{record.update_timestamp}</Badge></h6>
+            <h6>Created by: <Badge color={"secondary"} className={"float-right"}>{record.record_json.updated_by}</Badge></h6>
+            &nbsp;
+            <Table responsive>
+                <thead>
+                <tr>
+                    <td>Field</td>
+                    <td>Current Value</td>
+                    <td>Archived Value</td>
+                    <td></td>
+                </tr>
+                </thead>
+                <tbody>
+                    {modalItems}
+                </tbody>
+            </Table>
+        </section>;
+
+        // Set the state of the modal box to contain the body
+        this.setState({ modalBody: modalBody });
+        // Show the modal
+        this.showModal();
+    }
+
+    render() {
+        let modal = null, content = null;
+        // If there are no records, let's not render anything...
+        if(this.props.data.atd_txdot_change_log.length == 0) {
+            modal = null;
+            content = <h4>No changes found for this record.</h4>;
+        } else {
+            modal = <Modal isOpen={this.state.modal} className={this.props.className}>
+                <ModalHeader>Record Differences</ModalHeader>
+                <ModalBody>
+                    {this.state.modalBody}
+                </ModalBody>
+                <ModalFooter>
+                    <Button color="secondary" onClick={this.closeModal}>Close</Button>
+                </ModalFooter>
+            </Modal>;
+            content = <>
+                <h4>Record History</h4>
+
+                <Table responsive>
+                    <thead>
+                    <tr>
+                        <td>ChangeLog ID</td>
+                        <td>Date Archived</td>
+                        <td>Updated by</td>
+                        <td></td>
+                    </tr>
+                    </thead>
+                    <tbody>
+                    {this.props.data.atd_txdot_change_log.map(record => (
+                        <tr key={record.id}>
+                            <td>
+                                <Link to={`crashes/${record.id}`}>
```

Comment:
Like I said above, I don't think we need this column but also this link doesn't work and I don't know where it would take them in a fleshed out app.

Reply:
Ah, yes. This stuff I don't know what to do about.